id-authentication-default.properties

```diff
@@ -489,24 +492,24 @@ mosip.kernel.tokenid.partnercode.salt=${mosip.kernel.partnercode.salt}
 ## Partner Management Service allowed partner domains
 mosip.kernel.partner.allowed.domains=AUTH,DEVICE,FTM,MISP
 
-# IAM Adapter 
+# IAM Adapter
 mosip.iam.adapter.clientid=${mosip.ida.auth.clientId}
 mosip.iam.adapter.clientsecret=${mosip.ida.auth.secretKey}
 mosip.iam.adapter.appid=${mosip.ida.auth.appId}
 mosip.authmanager.client-token-endpoint=${mosip.kernel.authmanager.url}/v1/authmanager/authenticate/clientidsecretkey
 
-## IDA key generator 
+## IDA key generator
 keymanager.persistence.jdbc.driver=org.postgresql.Driver
 keymanager_database_url=jdbc:postgresql://${mosip.ida.database.hostname}:${mosip.ida.database.port}/mosip_ida
 keymanager_database_username=${mosip.ida.database.user}
 keymanager_database_password=${db.dbuser.password}
-mosip.kernel.keymanager.autogen.appids.list=ROOT,${application.id},${mosip.sign.applicationid}:${mosip.sign.refid},${application.id}:${mosip.kernel.zkcrypto.masterkey.reference.id},IDA_KYC_EXCHANGE,IDA_KEY_BINDING,IDA_VCI_EXCHANGE
+mosip.kernel.keymanager.autogen.appids.list=ROOT,${application.id},${mosip.sign.applicationid}:${mosip.sign.refid},${application.id}:${mosip.kernel.zkcrypto.masterkey.reference.id}
```

Comment:
@mahammedtaheer is the change valid? we still need ",IDA_KYC_EXCHANGE,IDA_KEY_BINDING,IDA_VCI_EXCHANGE" to be part of this list

Reply:
yes, it should be available in the list.

Reply:
Done